docker/oso-ops-base/rhel7/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM rhel7:latest
 
+ADD etc-pki-entitlement /etc-pki-entitlement
```

Comment:
This is only needed for Fedora. Build script should add it if needed. Old way used sed.